anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,193 @@
+
+
+def get_driver_binary():
+    value = data['dl']['selescrape_driver_binary_path']
+    binary_path = value.lower() if value else value
+    return binary_path
+
+
+def add_url_params(url, params):
+    encoded_params = urlencode(params)
+    url = url + '?' + encoded_params
+    return url
+
+
+def driver_select(): #
+    '''
+    it configures what each browser should do 
+    and gives the driver variable that is used 
+    to perform any actions below this function.
+    '''
+    browser = get_browser_config()
+    data_dir = get_data_dir()
+    executable = get_browser_executable()
+    driver_binary = get_driver_binary()
+    if driver_binary == None:
+        binary = None
+    else:
+        binary = driver_binary
+    if browser == 'firefox':
+        fireFoxOptions = webdriver.FirefoxOptions()
+        fireFoxOptions.headless = True
+        fireFoxOptions.add_argument('--log fatal')
+        if binary == None:  
+            driver = webdriver.Firefox(options=fireFoxOptions, service_log_path=os.path.devnull)
+        else:
+            try:
+                driver = webdriver.Firefox(options=fireFoxOptions, service_log_path=os.path.devnull)
+            except:
+                driver = webdriver.Firefox(executable_path=binary, options=fireFoxOptions, service_log_path=os.path.devnull)
+    elif browser == 'chrome':
+        from selenium.webdriver.chrome.options import Options
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")
+        chrome_options.add_argument("--disable-gpu")
+        profile_path = os.path.join(data_dir, 'Selenium_chromium')
+        log_path = os.path.join(data_dir, 'chromedriver.log')
+        chrome_options.add_argument(f'--log-path {log_path}')
+        chrome_options.add_argument(f"--user-data-dir={profile_path}")
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--window-size=1920,1080")
+        chrome_options.add_argument(f'user-agent={get_random_header()}')
+        if binary == None:
+            if executable == None:
+                driver = webdriver.Chrome(options=chrome_options)
+            else:
+                from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+                cap = DesiredCapabilities.CHROME
+                cap['binary_location'] = executable
+                driver = webdriver.Chrome(desired_capabilities=cap, options=chrome_options)
+        else:
+            if executable == None:
+                driver = webdriver.Chrome(options=chrome_options)
+            else:
+                from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+                cap = DesiredCapabilities.CHROME
+                cap['binary_location'] = executable
+                driver = webdriver.Chrome(executable_path=binary, desired_capabilities=cap, options=chrome_options)
+    return driver
+
+
+def status_select(driver, url, status='hide'):
```

Comment:
Can you explain what this does?

I can't treat it like it isnt here.

And improve these docs

Reply:
for now it is there to make it easier to add the http status message into selescrape. since i couldnt add it in a pythonic way i left it to stay there for me to add it in the future

Reply:
I don't get it. What http status? For what?

Reply:
if you want to make an if statement for the status code that is being returned.
like, if 404 then raise exception etc